ddclient

```diff
@@ -1991,74 +2091,67 @@ sub geturl {
     if ( $force_ssl || ($globals{'ssl'} and (caller(1))[3] ne 'main::get_ip') ) {
         $use_ssl      = 1;
         $default_port = 443;
-		load_ssl_support;
```

Comment:
This `load_ssl_support` is deleted here and moved down to where it is needed.  IPv6 only works with cURL so this should be loaded only if required for IPv4 and only if cURL is not being used for HTTP.